gramex/handlers/mlhandler.py

```diff
@@ -241,13 +241,26 @@ def get(self, *path_args, **path_kwargs):
         else:
             self._check_model_path()
             if '_download' in self.args:
-                self.set_header('Content-Type', 'application/octet-strem')
+                self.set_header('Content-Type', 'application/octet-stream')
                 self.set_header('Content-Disposition',
                                 f'attachment; filename={op.basename(self.store.model_path)}')
                 with open(self.store.model_path, 'rb') as fout:
                     self.write(fout.read())
             elif '_model' in self.args:
                 self.write(json.dumps(self.model.get_params(), indent=2))
+            elif len(self.request.files.keys()) and \
+                self.request.files['image'][0].content_type in \
+                    ['image/jpeg', 'image/jpg', 'image/png']:
+                if '_action' in self.args and self.args['_action'] == 'predict':
+                    import cv2
+                    import imutils
+                    data = imutils.resize(cv2.imdecode(np.fromstring(
+                        self.request.files['image'][0].body, np.uint8), cv2.IMREAD_UNCHANGED),
+                        width=224)
+                    data = cv2.resize(data, (224, 224))
```

Comment:
1. All of this logic should be in the wrapper class.
2. OpenCV should not be a dependency. For loading images from files / streams, use [`PIL.Image.open`](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.open)
3. For resizing, use `skimage.transform.resize` or `tf.image.resize` or `PIL.Image.resize`.
4. The target size after resizing should not be hardcoded to `[224, 224]` - there are other sizes in Keras apps too. For this, you can check the shape of the input tensor in the corresponding model with `model.input_shape`.